tools/ci/build-openmc.sh

```diff
@@ -3,6 +3,7 @@ set -ex
 
 # Build openmc
 cd openmc
+./tools/ci/gha-install-mcpl.sh
```

Comment:
Is this a hardcoded path? Will there ever be a time when this needs to be modified?

Reply:
Only if the corresponding path changes in OpenMC's repo

Reply:
This is within the OpenMC file structure, not anything the user would be defining?

Reply:
Correct!